custom_components/alexa_media/sensor.py

```
@@ -312,6 +306,10 @@ def __init__(
             if media_player_device_id
             else None
         )
+        if self._debug:
+            _LOGGER.debug(
+                "Coordinator init: %s",
+            )
```

Pylint (4.0.4)

[error] Not enough arguments for logging format string

(E1206)